### PR TITLE
Add better example for slew rate limiter

### PR DESCRIPTION
**Pull request by Oblarg**

### CODE REVIEW

File: source/docs/software/advanced-controls/filters/slew-rate-limiter.rst [hunk 1]
```diff
@@ -46,3 +46,26 @@ Once your filter has been created, using it is easy - simply call the ``calculat
 
     // Calculates the next value of the output
     filter.Calculate(input);
+
+Using a SlewRateLimiter with a Robot Drive
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (Daltz333) [inline source/docs/software/advanced-controls/filters/slew-rate-limiter.rst:51]:
```suggestion
Using a SlewRateLimiter with DifferentialDrive
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by Daltz333** [comment]
@prateekma could you verify C++?

File: source/docs/software/advanced-controls/filters/slew-rate-limiter.rst [hunk 2]
```diff
@@ -46,3 +46,26 @@ Once your filter has been created, using it is easy - simply call the ``calculat
...
+
+.. tabs::
+
+  .. code-tab:: java
+
+    // Ordinary call with no ramping applied
+    drivetrain.arcadeDrive(forward, turn);
+
+    // Slew-rate limits the forward/backward input, limiting forward/backward acceleration
+    driveTrain.arcadeDrive(filter.calculate(forward), turn);
```
Comment (prateekma) [inline source/docs/software/advanced-controls/filters/slew-rate-limiter.rst:63]: `driveTrain` -> `drivetrain`

File: source/docs/software/advanced-controls/filters/slew-rate-limiter.rst [hunk 3]
```diff
@@ -46,3 +46,26 @@ Once your filter has been created, using it is easy - simply call the ``calculat
...
+
+  .. code-tab:: c++
+
+    // Ordinary call with no ramping applied
+    drivetrain.ArcadeDrive(forward, turn);
+
+    // Slew-rate limits the forward/backward input, limiting forward/backward acceleration
+    driveTrain.ArcadeDrive(filter.Calculate(forward), turn);
```
Comment (prateekma) [inline source/docs/software/advanced-controls/filters/slew-rate-limiter.rst:71]: `driveTrain` -> `drivetrain`

**Review by prateekma — CHANGES_REQUESTED** [review]
I would also prefer if we showed users how to construct the filter here. The above example uses one templated on units::volts and that clearly won't work here.

**Comment by Daltz333** [comment]
@Oblarg update on prateeks review

**Comment by Oblarg** [comment]
I don't see any clear way to show how to construct it for C++ without breaking the flow of the article.  I added an explanatory note, instead.

**Comment by Daltz333** [comment]
I was talking about renaming driveTrain to drivetrain

**Comment by Oblarg** [comment]
Oh, I edited those in the wrong direction, it seems.

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**